chromium_src/third_party/blink/common/origin_trials/origin_trials.cc

```diff
@@ -48,16 +52,21 @@ bool IsTrialDisabledInBrave(base::StringPiece trial_name) {
 bool IsTrialDisabledInBrave(OriginTrialFeature feature) {
   // When updating also update the array in the overload above.
   // clang-format off
-  static const std::array<OriginTrialFeature, 9> kBraveDisabledTrialFeatures = {
-      OriginTrialFeature::kConversionMeasurement,
-      OriginTrialFeature::kDigitalGoods,
-      OriginTrialFeature::kFledge,
-      OriginTrialFeature::kParakeet,
-      OriginTrialFeature::kPrerender2,
-      OriginTrialFeature::kSignedExchangeSubresourcePrefetch,
-      OriginTrialFeature::kSubresourceWebBundles,
-      OriginTrialFeature::kTrustTokens,
-  };
+  static const std::array<OriginTrialFeature, 12> kBraveDisabledTrialFeatures =
```

Comment:
why not just `static const OriginTrialFeature kBraveDisabledTrialFeatures[] = {...};`?
`base::Contains` will work fine.

Reply:
Fixed in https://github.com/brave/brave-core/commit/60ef6240502d7381bc62b543fd9e680abc0d9346